app/main.py

```diff
@@ -16,4 +16,29 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [],
+    "immutable": []
+}
+
+variables = [
+    lucky_number,
+    pi,
+    one_is_a_prime_number,
+    name,
+    my_favourite_films,
+    profile_info,
+    marks,
+    collection_of_coins,
+]
+
+sorted_variables["mutable"].append(my_favourite_films)
```

Comment:
Why do you use append when you can just do that into "mutable": [my_favourite_films, .....]